ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml

```diff
@@ -21,15 +21,19 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: perfscale-rosa-network-perf-periodic
-  cron: 0 8 * * 5
+- as: perfscale-rosa-multiaz-cluster-density-v2-24nodes-periodic
+  cron: 0 8 * * 4
   steps:
     cluster_profile: aws-perfscale-qe
     env:
       CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
       OPENSHIFT_VERSION: "4.12"
+      REPLICAS: "24"
     test:
-    - chain: openshift-qe-data-path-tests
+    - ref: openshift-qe-cluster-density-v2
```

Comment:
think we would want to keep as data path tests here?

Reply:
Roshni suggested that we run only control-plane tests.